frontend/public/components/deployment.tsx

```diff
@@ -217,25 +218,31 @@ const ReplicaSetsTab: React.FC<ReplicaSetsTabProps> = ({ obj }) => {
 };
 
 const { details, editYaml, pods, envEditor, events } = navFactory;
-export const DeploymentsDetailsPage: React.FC<DeploymentsDetailsPageProps> = (props) => (
-  <DetailsPage
-    {...props}
-    kind={deploymentsReference}
-    menuActions={menuActions}
-    pages={[
-      details(DeploymentDetails),
-      editYaml(),
-      {
-        href: 'replicasets',
-        name: 'Replica Sets',
-        component: ReplicaSetsTab,
-      },
-      pods(),
-      envEditor(environmentComponent),
-      events(ResourceEventStream),
-    ]}
-  />
-);
+export const DeploymentsDetailsPage: React.FC<DeploymentsDetailsPageProps> = (props) => {
+  const ns = getActiveNamespace();
```

Comment:
you shouldn't be accessing store directly here in the component, Either use the redux hook `useSelector` or `connect` method. I would suggest using the hook here.

Reply:
used `useSelector`